digits/task.py

```diff
@@ -203,7 +203,9 @@ def run(self, resources):
         #https://docs.python.org/2/library/subprocess.html#converting-argument-sequence
         if platform.system() == 'Windows':
             args = ' '.join(args)
-        self.logger.info('Task subprocess args: {}'.format(args))
+            self.logger.info('Task subprocess args: "{}"'.format(args))
+        else:
+            self.logger.info('Task subprocess args: "%s"' % ' '.join(args))
```

Comment:
Thanks I always make this change locally to be able to copy/paste command lines into a command prompt! Otherwise you get the list notation (with brackets, commas et al.).